vendor/brave-ios/BATBraveRewards.h

```diff
@@ -13,8 +13,6 @@ NS_ASSUME_NONNULL_BEGIN
 NS_SWIFT_NAME(BraveRewardsConfiguration)
 @interface BATBraveRewardsConfiguration : NSObject <NSCopying>
 
-/// Whether or not rewards is being tested
-@property (nonatomic, getter=isTesting) BOOL testing;
```

Comment:
I don't see any changes made to ledgers testing variables, so this shouldn't be removed here or in BATBraveLedger

Reply:
Fixed